_data/members.yml

```diff
@@ -1,3 +1,6 @@
+- name: Albi Celaj
+ degree: PhD, Molecular Genetics
```

Comment:
YAML is finnicky. I believe degree needs to have another space in front of it.

Also, do you have a github id? Like Madeleine's below yours?